hack/markdown-preprocess

```diff
@@ -29,7 +29,7 @@ def process(infile):
     # Some options are the same between containers and pods; determine
     # which description to use from the name of the source man page.
     pod_or_container = 'container'
-    if '-pod-' in infile:
+    if '-pod-' in infile or '-kube-' in infile:
```

Comment:
The `kube` man pages tend to speak in terms of pods, so I choose to treat them as pod man pages for purposes of `<<container|pod>>` expansion.